deploy/ocp-templates/heketi-template.yaml

```diff
@@ -121,3 +123,6 @@ parameters:
   displayName: heketi fstab path
   description: Set the fstab path, file that is populated with bricks that heketi creates
   value: /var/lib/heketi/fstab
+- name: HEKETI_IGNORE_STALE_OPERATIONS
+  displayName: Heketi ignore stale operations
+  description: Whether to ignore the stale pending operations db entries in the database
```

Comment:
Should we be a little more verbose?

"This allows to control whether heketi should start up when when there are stale pending operation entries present in the database."

Maybe even more (not sure how long this can/should be):  " ... Setting this to true lets heketi ignore existing pending operations at startup. Setting it to false causes heketi to refuse to start if pending operations are found in the database."